tests/manage/z_cluster/test_mon_data_avail_warn.py

```diff
@@ -0,0 +1,118 @@
+"""
+Testing MON disk low threshold
+Automating BZ#1964055
+"""
+import logging
+import time
+import random
+import pytest
+
+from ocs_ci.framework.testlib import E2ETest, tier2, bugzilla, skipif_ocs_version
+from ocs_ci.ocs.resources import pod
+from ocs_ci.ocs.cluster import CephCluster
+from ocs_ci.utility import utils
+from ocs_ci.helpers.helpers import run_cmd_verify_cli_output
+
+log = logging.getLogger(__name__)
+
+SLEEP_TIMEOUT = 20
+TEMP_FILE = "tmp_f"
+DD_BLOCK_SIZE = 16
+DD_COUNT = 64
+
+
+@tier2
+@bugzilla("1964055")
+@pytest.mark.polarion_id("OCS-2733")
+@skipif_ocs_version("<4.9")
+class TestMonDataAvailWarn(E2ETest):
+    """
+    Testing MON disk low threshold.
+    Ceph health enters '' state once mon disk reaches >= 85%
+
+    """
+
+    mon_pod = None
+    mon_suffix = None
+    workloads_dir = None
+    dd_seek_count = 0
+
+    @pytest.fixture()
+    def workloads_dir_setup(self, request):
+        """
+        Setting up the environment for the test
+
+        """
+
+        self.mon_pod = random.choice(pod.get_mon_pods())
+        self.mon_suffix = self.mon_pod.get().get("metadata").get("labels").get("mon")
+        self.workloads_dir = f"/var/lib/ceph/mon/ceph-{self.mon_suffix}/workloads"
```

Comment:
The mon host path for LSO will be different, I think the path on LSO is  /var/lib/rook/mon-a

Reply:
Done.
Added code to handle case of LSO cluster